Directory.Build.targets

```diff
@@ -38,6 +38,7 @@
     <Product>Microsoft%AE .NET</Product>
     <!-- Use the .NET product branding version for informational version description -->
     <InformationalVersion Condition="'$(InformationalVersion)' == '' and '$(VersionSuffix)' == ''">$(ProductVersion)</InformationalVersion>
+    <InformationalVersion Condition="'$(InformationalVersion)' == '' and '$(PreReleaseVersionLabel)' == 'servicing'">$(ProductVersion)</InformationalVersion>
```

Comment:
Any idea why `VersionSuffix` is even set during servicing builds? Arcade unsets it here: https://github.com/dotnet/arcade/blob/cb64095ec45ac34378a1a26db9932a3b561f9e4e/src/Microsoft.DotNet.Arcade.Sdk/tools/Version.BeforeCommonTargets.targets#L85-L90

Reply:
We have `'$(PreReleaseVersionLabel)' == or != 'servicing'` conditions in https://github.com/dotnet/runtime/blob/release/7.0/eng/packaging.targets which takes care of assemblies under std/libs, but not under src/coreclr etc. which is likely why we have these differences. Such conditions are all over the place:

```sh
$ git checkout release/7.0
$ git grep PreReleaseVersionLabel

eng/Versions.props:    <PreReleaseVersionLabel>servicing</PreReleaseVersionLabel>
eng/packaging.targets:                                       '$(PreReleaseVersionLabel)' != 'servicing' and
eng/packaging.targets:  <PropertyGroup Condition="'$(PreReleaseVersionLabel)' == 'servicing'">
eng/packaging.targets:          Condition="'$(SkipValidateAssemblyVersion)' != 'true' and '$(_AssemblyInTargetingPack)' == 'true' and '$(PreReleaseVersionLabel)' == 'servicing'"
eng/packaging.targets:          Condition="'$(PreReleaseVersionLabel)' == 'servicing' and '$(DotNetBuildFromSource)' != 'true'"
src/coreclr/.nuget/Directory.Build.props:  <PropertyGroup Condition="'$(PreReleaseVersionLabel)' == 'servicing'">
src/coreclr/.nuget/Directory.Build.targets:    <StableVersion Condition="'$(StabilizePackageVersion)' == 'true' and '$(StableVersion)' == '' and '$(PreReleaseVersionLabel)' != 'servicing'">$(PackageVersion)</StableVersion>
src/coreclr/System.Private.CoreLib/System.Private.CoreLib.csproj:    <InformationalVersion Condition="'$(PreReleaseVersionLabel)' == 'servicing'">$(ProductVersion)</InformationalVersion>
src/libraries/Microsoft.Internal.Runtime.AspNetCore.Transport/src/Microsoft.Internal.Runtime.AspNetCore.Transport.proj:  <PropertyGroup Condition="'$(PreReleaseVersionLabel)' == 'servicing'">
src/libraries/Microsoft.Internal.Runtime.WindowsDesktop.Transport/src/Microsoft.Internal.Runtime.WindowsDesktop.Transport.proj:  <PropertyGroup Condition="'$(PreReleaseVersionLabel)' == 'servicing'">
src/mono/System.Private.CoreLib/System.Private.CoreLib.csproj:    <InformationalVersion Condition="'$(PreReleaseVersionLabel)' == 'servicing'">$(ProductVersion)</InformationalVersion>
```

Ideally all shipping packages in this repo should use `eng/packaging.targets`, but that infra isn't consolidated (yet).